l1-contracts/src/UniFiAVSManager.sol

```diff
@@ -508,6 +695,17 @@ contract UniFiAVSManager is UniFiAVSManagerStorage, IUniFiAVSManager, UUPSUpgrad
         return address(AVS_DIRECTORY);
```

Comment:
Inconsistent getter naming, the previous function is called `getRestakeableStrategies` and this one just `avsDirectory`